pkg/glusterfs/controllerserver.go

```diff
@@ -99,6 +111,42 @@ func (cs *ControllerServer) ParseCreateVolRequest(req *csi.CreateVolumeRequest)
 	return &reqConf, nil
 }
 
+func validateThinArbiter(req *csi.CreateVolumeRequest) error {
+	if _, ok := req.Parameters["arbiterPath"]; ok {
```

Comment:
@JohnStrunk I have added `arbiterPath` to storage class to get the arbiter path  which will be like `<host:port/path/to/brick>` i thought  this cant be handled  with arbiterzones  as  you  suggested [here](https://github.com/gluster/gluster-csi-driver/pull/30/files#diff-c6b9b1189a45890edc0dcc6ed45f0205R102)

any suggestions from your side on this?